server/utils/placementRequests/adminIdentityBar.test.ts

```diff
@@ -27,23 +40,54 @@ describe('adminIdentityBar', () => {
       ])
     })
 
-    it('should return actions to create a booking and withdraw a placement request if the status is not `matched`', () => {
-      const placementRequestDetail = placementRequestDetailFactory.build({ status: 'notMatched' })
+    describe('if ENABLE_V2_MATCH is false', () => {
+      it('should return actions to create a booking and withdraw a placement request if the status is not `matched`', () => {
+        const placementRequestDetail = placementRequestDetailFactory.build({ status: 'notMatched' })
 
-      expect(adminActions(placementRequestDetail)).toEqual([
-        {
-          href: adminPaths.admin.placementRequests.bookings.new({ id: placementRequestDetail.id }),
-          text: 'Create placement',
-        },
-        {
-          href: applyPaths.applications.withdraw.new({ id: placementRequestDetail.applicationId }),
-          text: 'Withdraw request for placement',
-        },
-        {
-          href: adminPaths.admin.placementRequests.unableToMatch.new({ id: placementRequestDetail.id }),
-          text: 'Mark as unable to match',
-        },
-      ])
+        expect(adminActions(placementRequestDetail)).toEqual([
+          {
+            href: adminPaths.admin.placementRequests.bookings.new({ id: placementRequestDetail.id }),
+            text: 'Create placement',
+          },
+          {
+            href: applyPaths.applications.withdraw.new({ id: placementRequestDetail.applicationId }),
+            text: 'Withdraw request for placement',
+          },
+          {
+            href: adminPaths.admin.placementRequests.unableToMatch.new({ id: placementRequestDetail.id }),
+            text: 'Mark as unable to match',
+          },
+        ])
+      })
+    })
+
+    describe('if ENABLE_V2_MATCH is true', () => {
+      const originalFlagValue = config.flags.v2MatchEnabled
+      beforeEach(() => {
+        config.flags.v2MatchEnabled = 'true'
+      })
+      afterEach(() => {
+        config.flags.v2MatchEnabled = originalFlagValue
+      })
+
+      it('should return the v2 path for "Create placement"', () => {
+        const placementRequestDetail = placementRequestDetailFactory.build({ status: 'notMatched' })
+
+        expect(adminActions(placementRequestDetail)).toEqual([
```

Comment:
could we use an "inclusion assertion" here (like https://github.com/ministryofjustice/approved-premises-ui/blob/98e2b9769263bb0896d0e94cc2e9f51861a606b5/server/testutils/jest.setup.ts#L73) to be more granular in describing which actions are included in the list of links? The problem with asserting equality on the whole list is that we don't get a helpful error if the entire set of links isn't as expected...

Reply:
yeah absolutely - this slipped my mind